estate/views/estate_property_views.xml

```diff
@@ -0,0 +1,238 @@
+<?xml version="1.0"?>
+<odoo>
+
+    <!--Actions-->
+
+    <record id="estate_property_action" model="ir.actions.act_window">
+        <field name="name">Properties</field>
+        <field name="res_model">estate.property</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
+    <record id="estate_property_type_action" model="ir.actions.act_window">
+        <field name="name">Property Types</field>
+        <field name="res_model">estate.property.type</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
+    <record id="estate_property_tag_action" model="ir.actions.act_window">
+        <field name="name">Property Types</field>
+        <field name="res_model">estate.property.tag</field>
+        <field name="view_mode">tree,form</field>
+    </record>
```

Comment:
for each model we create separate files like `estate_property_tag_view` and `estate_property_type_view` to organise code better and put there all views related to the model

Reply:
makes sense